_pages/production.md

```diff
@@ -2,81 +2,150 @@
+6.  Select the authentication protocol that will be used by the service provider. 
 
-Before you can deploy your application to the production environment, you will need to create a separate app on our dashboard that contains your production certificate, urls and logo. Here are the steps to complete your production configuration app:
-1. Create a new app on the dashboard <https://dashboard.int.identitysandbox.gov/>
-2. Indicate that the app configuration is for a Production application 
-3. Enter the name of the application **as it appears in your IAA** in the "App Name" field - Do **NOT** use environment names such as "Prod" or "Production"
-4. Enter the name of the application **as you want it to appear to users** in the "Friendly Name" field
-5. Enter the production URLs and configuration into the app
+7.  Choose the level of service and default **Authentication Assurance Level (AAL)** from the drop down menus and select the correct attribute bundle. 
 
-**Please note**: The following items are required to promote your app to production:
+8.  Enter the issuer, a unique string to identify the app in the **Identity Provider (IdP).** 
 
-* All production urls should have .gov .mil or a dedicated .com address and point to an ATO-ed environment.
-* You must include a logo for your application. You can find the [logo guidelines here](https://developers.login.gov/design-guidelines/#agency-logo-guidelines).
-* If this is a SAML integration (Not OpenID Connect), then please ensure that:
-  * Assertion Consumer Logout Service URL is defined.
-  * SAML Assertion Encryption is enabled.
-    * If you are using a service which does not support SAML encryption, then please [submit a support request](https://logingov.zendesk.com/) for further guidance.
-* If this is an integration requesting identity proofed attributes, you **must** include a Failure to Proof URL. This is the URL users will be sent to if they fail to complete the identity verification process so you can communicate alternative methods of accessing your application.
+9.  Upload your logo and public certificate file using the "Choose a file" buttons.
 
-## Request deployment
+10. Enter the push notification URL and redirect URIs. You can enter additional redirect URIs by selecting the "Add another URI" button.
+
+11. Specify the sign-in, sign-up, and forgot password help text users will encounter in your app.   
+
+12. Once all fields are complete select the "Create test app" button. 
+
+If you encounter errors or have questions after completing these steps, please [submit a technical support ticket](https://zendesk.login.gov/).
+
+Production endpoints
+--------------------
+
+You will need to configure your application to point to the following endpoint: 
+
+-   OpenID Connect: [https://secure.login.gov/openid_connect/authorize](https://secure.login.gov/openid_connect/authorize)
+
+-   SAML: [https://secure.login.gov/api/saml/auth2022](https://secure.login.gov/api/saml/auth2022)
```

Comment:
I think our style has to been to use monospace fonts for URLs that are meant as configuration:
```suggestion
-   OpenID Connect: `https://secure.login.gov/openid_connect/authorize`

-   SAML: `https://secure.login.gov/api/saml/auth2022`
```